lexos/processors/analyze/information.py

```diff
@@ -61,52 +55,42 @@ def __init__(self, word_lists, l_files):
             elif file_sizes[file] < mid - 1.5 * iqr:
                 file_anomaly_iqr.update({file.name: 'small'})
 
-        # pack the data
+        # Pack the data
         self.NumFile = num_file  # number of files
-        # an array of the total word count of each file
-        self.FileSizes = file_sizes
+        self.FileSizes = file_sizes  # array of word count of each file
         self.Average = average_file_size  # average file size
-        self.StdE = std_err_file_size  # standard error of file size
-        # an array contain dictionary map anomaly file to
-        # how they are different from others(too large or too small)
-        # analyzed in using standard error
+        # standard deviation of this population
+        self.stdDeviation = std_dev_file_size
```

Comment:
no upper case variable, please fix all of them.

search using the following regex: `self\..*[A-Z].*`